src/tests/ftest/util/command_utils.py

```diff
@@ -83,7 +83,7 @@ def update(self, value, name=None):
 
 
 class FormattedParameter(BasicParameter):
-    # pylint: disable=too-few-public-methods
+    # pylint: disable=too-few-public-methods, pylint-no-member
```

Comment:
Warning: By disabling `pylint-no-member` here it applies to the entire class and the warning will not trigger in the future if a change in the code resulted in this issue.

Assuming it's complaining about the addition to the `__str__()` function, this slight change *may* avoid the warning:
```
            if isinstance(self.value, dict):
                return " ".join(
                    [self._str_format.format("{} \"{}\"".format(key, self.value[key]))
                     for key in self.value])
```

Reply:
Pushed a change to change to your suggestion.